pmlc/target/intel_gen/passes.td

```diff
@@ -11,6 +11,11 @@ def IntelGenLowerAffine : FunctionPass<"intel-gen-lower-affine"> {
   ];
 }
 
+def AffinePack : FunctionPass<"affine-pack"> {
```

Comment:
Suggest that if this is intel-gen specific, prefix that in the command line. If it's not, then move this to `pxa`.

Reply:
Done.  Although really some of these passes belong in a generic GPU directory (since this applies to other GPUs), leaving as intel-gen for now